sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala

```diff
@@ -492,15 +498,16 @@ class RocksDBStateStoreSuite extends StateStoreSuiteBase[RocksDBStateStoreProvid
     }
   }
 
-  testWithColumnFamilies("rocksdb range scan multiple non-contiguous ordering columns",
-    TestWithBothChangelogCheckpointingEnabledAndDisabled ) { colFamiliesEnabled =>
+  testWithColumnFamiliesAndEncodingTypes(
+    "rocksdb range scan multiple non-contiguous ordering columns",
+    TestWithBothChangelogCheckpointingEnabledAndDisabled ) { (colFamiliesEnabled, encodingType) =>
     val testSchema: StructType = StructType(
       Seq(
-        StructField("ordering-1", LongType, false),
+        StructField("ordering1", LongType, false),
         StructField("key2", StringType, false),
-        StructField("ordering-2", IntegerType, false),
```

Comment:
can we add a test to verify the behavior if `-` is used within the state var names since its not supported in Avro ?